core/object/class_db.h

```diff
@@ -372,17 +372,17 @@ class ClassDB {
 	static uint64_t get_native_struct_size(const StringName &p_name); // Used for asserting
 };
 
-#ifdef DEBUG_METHODS_ENABLED
-
-#define BIND_CONSTANT(m_constant) \
-	::ClassDB::bind_integer_constant(get_class_static(), StringName(), #m_constant, m_constant);
-
 #define BIND_ENUM_CONSTANT(m_constant) \
 	::ClassDB::bind_integer_constant(get_class_static(), __constant_get_enum_name(m_constant, #m_constant), #m_constant, m_constant);
 
 #define BIND_BITFIELD_FLAG(m_constant) \
 	::ClassDB::bind_integer_constant(get_class_static(), __constant_get_bitfield_name(m_constant, #m_constant), #m_constant, m_constant, true);
 
+#ifdef DEBUG_METHODS_ENABLED
+
+#define BIND_CONSTANT(m_constant) \
+	::ClassDB::bind_integer_constant(get_class_static(), StringName(), #m_constant, m_constant);
+
```

Comment:
Given this is the same with or without `DEBUG_METHODS_ENABLED` it could stay outside the `#ifdef` too.

Reply:
Sorry, that's probably because I made mistake when I tried to fix conflicts. 
And unfortunately, my branch seems to be broken now (cannot checkout). I will close this PR and make new one later if necessary.